site/docs/README.md

```diff
@@ -10,3 +10,19 @@ In order to speed up the deployment there is a GCR image
 python requirements pre-installed.  This cuts the deployment from several
 minutes to around twenty seconds.  To rebuild and deploy the image use the
 `deploy-builder.sh` script.
+
+# Update Hugo version
+
+The Hugo version is defined by variable `HUGO_EXTENDED_VERSION` in `util/build_docs.py`.
+To ensure syntax highlighting is working correctly the CSS stylesheet must be updated following a version update.
+
+## Update CSS stylesheet
+
+Setting the option `noClasses = false` for `[markup.highlight]` in `site/docs/config.toml` requires a stylesheet to be available.
+This option is used in order to have two styles for light and dark mode of the documentation site.
+The stylesheet is stored in `site/docs/assets/scss/_chroma.scss`.
+Update the style if the Hugo version is changed:
+
+- Replace the content of `[data-user-color-scheme='light']` with the output of `hugo gen chromastyles --style=colorful`, but keep the first line containing the setting of the background.
+
+- Replace the content of `[data-user-color-scheme='dark']` with the output of `hugo gen chromastyles --style=dracula`, but keep the first line containing the setting of the background.
```

Comment:
Instead of modifying the style, can you keep it unmodified and add do an override in another CSS class?

Reply:
I could, but then the enclosure with `[date-user-color-scheme='*'] { * }` needs to be added as well, so the command to produce the output would be slightly more complicated, so I thought this might be easier. But I understand that the current procedure requires more manual work which might not be optimal.

Reply:
OK, let's leave it as-is for now, and I'll have a look when I find time.